src/System.CommandLine.Tests/ParseResultValueTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.CommandLine.Parsing;
+using System.Linq;
+using FluentAssertions;
+using Xunit;
+
+namespace System.CommandLine.Tests;
+
+public class ParseResultValueTests
+{
+    [Fact]
+    public void Symbol_found_by_name()
+    {
+        var option1 = new CliOption<string>("--opt1");
+        var option2 = new CliOption<string>("--opt2");
+
+        var rootCommand = new CliRootCommand
+                    {
+                        option1,
+                        option2
+                    };
+
+        var parseResult = CliParser.Parse(rootCommand, "--opt1 Kirk");
+
+        var symbol1 = parseResult.GetSymbolByName("--opt1");
+        var symbol2 = parseResult.GetSymbolByName("--opt2");
+        symbol1.Should().Be(option1);
+        symbol2.Should().Be(option2);
```

Comment:
```suggestion
        using (new AssertionScope())
        {
            symbol1.Should().Be(option1);
            symbol2.Should().Be(option2);
        }
```
This could be a nice place to use an assertion scope if we are going to have multiple assertions in one test?

Reply:
Great idea. I think this case warrants to two options as one state.